platforms/hermes-python/hermes_python/ontology/dialogue/session.py

```diff
@@ -115,13 +118,16 @@ def __eq__(self, other):
 
 class SessionStartedMessage(object):
     def __init__(self, session_id, custom_data, site_id, reactivated_from_session_id):
+        # type: (str, Optional[str], str, Optional[str]) -> None
         """
         A message that the handler receives from the Dialogue Manager when a session is started.
 
         :param session_id: Session identifier that was started.
         :param custom_data: Custom data provided in the start session request on.
         :param site_id:  Site where the user interaction is taking place.
-        :param reactivated_from_session_id: This field is left blank voluntarily.
+        :param reactivated_from_session_id: This optional field indicates this session is a reactivation of a previously
```

Comment:
note that this is not supported by the dialogue yet

Reply:
ouch ! removing this from the API then.